src/algokit_utils/network_clients.py

```diff
@@ -82,6 +82,18 @@ def is_localnet(client: AlgodClient) -> bool:
     return params.gen in ["devnet-v1", "sandnet-v1", "dockernet-v1"]
 
 
+def is_mainnet(client: AlgodClient) -> bool:
+    """Returns True if client genesis is `mainnet-v1`"""
+    params = client.suggested_params()
+    return bool(params.gen == "mainnet-v1")
```

Comment:
The bool conversion can be removed here and in `is_testnet`?

Reply:
Yes cetainly but I got the mypy error.

Reply:
No problem, this seems like the best solution then